src/main/java/frc/robot/subsystems/LimelightSubsystem.java

```diff
@@ -1,20 +1,18 @@
 package frc.robot.subsystems;
 
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
-import edu.wpi.first.wpilibj2.command.WaitCommand;
 import edu.wpi.first.wpilibj.smartdashboard.*;
 import edu.wpi.first.networktables.NetworkTableInstance;
 import edu.wpi.first.networktables.NetworkTable;
-import frc.robot.Constants.GameConstants;
 //constants
 import frc.robot.Constants.LimelightConstants; 
 
 public class LimelightSubsystem extends SubsystemBase {
     private NetworkTable table;
-
+  
     public LimelightSubsystem() {
         this.table = NetworkTableInstance.getDefault().getTable("limelight");
-        setPipeline(1);
+        setPipeline(3);
```

Comment:
For situations like this, instead of directly using these "magic numbers", it's better to make a quick enum (or something like that) representing the different options. For example, it might look something like this:

```java
enum Pipeline {
    DRIVER_VIEW(0), CONE_PEG(1), CUBE_PLATFORM(2);

    private int pipelineId;
}
```

Doing it this way means anyone who looks at something like `setPipeline(Pipeline.DRIVER_VIEW)` can directly tell what pipeline is being set and minimizes the chance for errors. You also can't pass in an invalid value (like 17), although you do need to check for null.